src/main/java/com/jcabi/log/VerboseProcess.java

```diff
@@ -433,6 +485,9 @@ public Void call() throws Exception {
                     new OutputStreamWriter(this.output, VerboseProcess.UTF_8)
                 );
                 try {
```

Comment:
@dean-e-clark for so complex logic added in this PR I'd expect at least dozen of unit tests, but I don't see any :disappointed:

Reply:
have you read this: http://www.yegor256.com/2017/03/24/tdd-that-works.html? 

What about writing unit tests only when bugs will appear?